components/raftstore/src/store/metrics.rs

```diff
@@ -302,6 +314,24 @@ lazy_static! {
             "Bucketed histogram of proposals' write db end duration.",
             exponential_buckets(0.00001, 2.0, 26).unwrap()
         ).unwrap();
+    pub static ref STORE_KNOW_PERSIST_DURATION_HISTOGRAM: Histogram =
```

Comment:
How about removing the `know_` prefix? It seems redundant.

Reply:
Done.